tests/UI/webservices/country/countryXml.ts

```diff
@@ -0,0 +1,65 @@
+import BaseXml from '@webservices/baseXml';
+import xmlHelper from '@utils/xml';
+
+export default class CountryXml extends BaseXml {
```

Comment:
```suggestion
export default class Xml extends BaseXml {
```

Reply:
I didn't want to have all classes with same name that's why